catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -90,6 +94,9 @@ public UpdateRequest process(UpdateRequest input)
                 counter.getOrDefault(Validation.VALIDATION_WARNINGS, 0),
                 counter.getOrDefault(Validation.VALIDATION_ERRORS, 0));
 
+        INGEST_LOGGER.info("{} metacards were removed for failing an enforced validator.",
```

Comment:
I think if we don't want this at `info` we might as well remove it - the individual enforcements already log at the debug level.